src/apps/Agreement/DisputableApps/DisputableApps.js

```diff
@@ -45,18 +43,21 @@ function renderEntry({ allowedActions }, compactMode) {
         ${!compactMode && `height: ${4 * GU}px;`}
       `}
     >
-      {/* TODO: Replace with LocalLabelAppBadge */}
-      <KnownAppBadge appName="voting.aragonpm.eth" label="Voting" />
+      {app && <LocalLabelAppBadge app={app} apps={[]} noIdentifier />}
     </div>,
-    <React.Fragment>{allowedActions.join(', ')}</React.Fragment>,
+    <React.Fragment>
+      {allowedActions.length > 0
+        ? allowedActions.join(', ')
+        : 'No assigned actions to display'}
```

Comment:
Not sure on this language, I wasn't clear on the context for the case (other than our currently missing roles on disputable voting)